packages/main/src/pages/static/contact-us/faq/detailsArray.js

```diff
@@ -1,6 +1,5 @@
 const Topics = [
   {
-    topic: "topic_adding_removing_users",
```

Comment:
isn't this necessary for the translation?

Reply:
it has been changed please review